recipe/meta.yaml

```diff
@@ -74,6 +74,8 @@ requirements:
     - xz {{ xz }}
     - zlib {{ zlib }}
     - zstd {{ zstd }}
+  run_constrained:
```

Comment:
I think `run` would make more sense. I'm also curious to know why it doesn't use libpoppler-cpp which has a stable ABI.

Reply:
Yes, run + pin_compatible. poppler doesn't have run_exports set. 
Then it should be rebuilt yet another time against our newer poppler in main (24.0.9)